setup.py

```diff
@@ -8,7 +8,7 @@ def readme():
 
 setup(
     name="zookeeper",
-    version="0.5.5",
+    version="1.0.0-dev",
```

Comment:
To make sure we follow https://www.python.org/dev/peps/pep-0440/
```suggestion
    version="1.0.dev1",
```